src/rpc/rawtransaction.cpp

```diff
@@ -203,6 +204,123 @@ static UniValue getrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+static UniValue searchrawtransactions(const JSONRPCRequest& request) {
```

Comment:
In commit "Add searchrawtransactions RPC" (226eeea9736127269058fbbf7816c7100f90974d)

Should add a basic python functional test for the new RPC method.